contracts/Trader.sol

```diff
@@ -20,7 +21,7 @@ contract Trader is ITrader {
     // EIP712 Types
     bytes32 private constant ORDER_TYPE =
         keccak256(
-            "Order(address maker,uint256 amount,int256 price,uint256 filled,bool side,uint256 expiration,uint256 creation,address targetTracer)"
+            "Order(address maker,uint256 amount,int256 price,uint256 filled,bool side,uint256 expiration,uint256 created,address market)"
         );
```

Comment:
I guess this should also change `expiration` to `expires`

Reply:
Consult 55421b0.